src/generate-grpc-web.ts

```diff
@@ -383,7 +383,12 @@ function createInvokeMethod(ctx: Context) {
             },
           });
           observer.add(() => {
-            if (!observer.closed) return client.close()
+           ${
+             !useAbortSignal
+               ? `return client.close();`
+               : `if (!abortSignal || !abortSignal.aborted) 
```

Comment:
Ah gotcha; @hersentino it looks like we don't have an `integration/` test that generates the `!abortSignal || !abortSignal.aborted` case? If not, do you mind adding one? Just like copy/paste the `grpc-web` directory to like `grpc-web-abort-signal` and add the abort signal flag to `parameters.txt` I think? 

No need to actually have a runtime test suite that covers this behavior, b/c I assume that is hard to test (unless it's easy to test, then feel free! :-), but at least having some generated code checked in that shows this code being generated will help us keep the behavior from regressing.

Thanks!

Reply:
>  it looks like we don't have an integration/ test that generates the !abortSignal || !abortSignal.aborted case?

We have one, it is located in `integration/grpc-web-no-streaming-observable`, but since it does not contain streaming-observable and my fix is specifically related to streaming, there was no difference when I ran `bin2ts`.

> do you mind adding one?

@stephenh Tell me what you prefer, keeping it in `integration/grpc-web-no-streaming-observable` or creating a new one?

Reply:
It was quick, so I added it

Reply:
That's great, thanks @hersentino !